addons/account/models/account_move.py

```diff
@@ -3350,14 +3374,11 @@ def _compute_tax_audit(self):
             audit_str = ''
             for tag in record.tag_ids:
 
-                caba_origin_inv_type = record.move_id.type
-                caba_origin_inv_journal_type = record.journal_id.type
-
                 if record.move_id.tax_cash_basis_rec_id:
                     # Cash basis entries are always treated as misc operations, applying the tag sign directly to the balance
                     type_multiplicator = 1
                 else:
-                    type_multiplicator = (record.journal_id.type == 'sale' and -1 or 1) * (self._get_refund_tax_audit_condition(record) and -1 or 1)
+                    type_multiplicator = (record.journal_id.type == 'sale' and self._get_not_entry_condition(record) and -1 or 1) * (self._get_refund_tax_audit_condition(record) and -1 or 1)
```

Comment:
Actually, this (and the _get_not_entry_condition function and override) are part of another fix than the reverse needing to create a refund, so we should make a distinct commit for it as well (and update PR description)